### DISCUSSION TITLE
Net Energy cost calculated sensor

### DISCUSSION THREAD

**Discussion by usbrit** [BottlecapDave/HomeAssistant-OctopusEnergy#915] [Q&A]
I have had the below sensor working correctly for a few months, however it stopped working this week.

Any advice as to how to fix would be appreciated.

`- platform: template
  sensors:
    net_electric_cost:
      friendly_name: "Net Electricity"
      unit_of_measurement: "£"
      value_template: "{{ ((states('sensor.octopus_energy_cost_tracker_solorman_import_cost') | float) - (states('sensor.octopus_energy_cost_tracker_electricity_export_today') | float)) | round(2) }}"`

**Comment by BottlecapDave** [comment]
Hello. I'm not sure why this sensor would stop working. I assume you haven't renamed the entity that is being tracked and been hit by this [bug](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/825)? Do you have any errors or warning in your logs? If not, the only suggestion I have is to reload the integration or restart your HA instance.
